### ISSUE TITLE
Header Component

### ISSUE THREAD

**Issue by DomVinyard** [Sigma-Labs-XYZ/explain-ai#15]

**Comment by s-dilip** [comment]
When the dropdown/ select is opened on mobile, does the list of options that appear need to be styled as well? Or can they have the default dropdown style that comes standard with vanilla `select` tag?

**Comment by s-dilip** [comment]
@DomVinyard Is it ok to create custom components to add inside the components mentioned in the ticket? So me and @yuna-iwata were thinking of creating a component for the age toggle and then importing and adding it to the Header component.

**Comment by bp289** [comment]
Hi everyone, when we looked at testing for responsive design we couldnt figure out how to check css style elements with jest. I did some research on this and i found that its best to use cypress to achieve this. I believe this is part of our learning objectives so i thought it would be useful to provide this information for everybody:
https://medium.com/xord/responsiveness-testing-using-cypress-3d15824d163e

**Comment by mdapim** [comment]
Hi @DomVinyard , @s-dilip , @yuna-iwata we've got a slight issue in which the header component looks squashed might be an issue on our end, are we okay to mess around with it to get it working, mainly the styling, and would this need to be on a new branch after we're done with it?
- can add it to break header component down, etc.

**Comment by s-dilip** [comment]
@mdapim Yes, you can change around styling. I do wonder why it looks squashed though. The styling seems fine on the deployed version. Does it look the squashed for @bp289 too?

**Comment by mdapim** [comment]
Thanks @s-dilip we'll finish everything first and have a look after, it still looks the same on both, what about you guys @Ademsk1 @Mariam-mohamed110?, might just be us then, if it looks good on the deployed ones

**Comment by Ademsk1** [comment]
I'm not getting that styling issue, but I do get something strange when I move to the phone screen: 
<img width="632" alt="Screenshot 2023-02-28 at 17 43 35" src="https://user-images.githubusercontent.com/85499621/221934574-0216f846-4d81-4ed2-9c79-be059276461c.png">

**Comment by Ademsk1** [comment]
Get this on the deployed website too. 
<img width="195" alt="Screenshot 2023-02-28 at 17 46 13" src="https://user-images.githubusercontent.com/85499621/221935239-7ee19586-3835-4905-93d6-06a86dd0db5b.png">

**Comment by mdapim** [comment]
It might be an issue with us then, thanks, we'll leave the code and try figure it out, thanks @Ademsk1
